src/screens/OtherProfile/index.js

```diff
@@ -61,8 +61,9 @@ import {trimString} from '../../utils/string/TrimString';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
 import StorageUtils from '../../utils/storage';
 import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
+import useDiscovery from '../DiscoveryScreenV2/hooks/useDiscovery';
 
-const {width, height} = Dimensions.get('screen');
+const {width} = Dimensions.get('screen');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `height` variable from `Dimensions.get('screen')` has been removed. If it's not used elsewhere in the code, this change is fine. However, if it's used and not defined elsewhere, this could lead to a reference error.
```diff
- const {width, height} = Dimensions.get('screen');
+ const {width} = Dimensions.get('screen');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->